app/models/claim.rb

```diff
@@ -74,8 +90,17 @@ def data_for_calculation
       work_items: work_items_for_calculation,
       disbursements: disbursements_for_calculation,
       letters_and_calls: letters_and_calls_for_calculation,
+      youth_court: data['youth_court'] == 'yes',
+      # TODO: CRM457-2306: Amend these as the fields will exist
+      claimed_youth_court_fee_included: data['include_youth_court_fee'],
+      plea_category: data['plea_category'],
+      # TODO: CRM457-2306: Remove these as the fields will exist
+      # :nocov:
+      assessed_youth_court_fee_included: data['youth_court_fee_adjustment_comment']&.present?,
```

Comment:
This is backwards. If there is an adjustment comment it will be because a caseworker has _removed_ the youth court fee.

The value of assessed_youth_court_fee_included is always available in `data['youth_court_fee_included']` so you can just use that.

Reply:
Both of these will be picked up in 2306, but yeah you're right

Reply:
Just to clarify the intention is to put the correct input for this field as a part of the next PR (that handles adjustments) - this is essentially acting as a placeholder as youth_court_adjustment_comment can't exist yet. We probably should have entered just "false" for now, with a comment explaining what we intend to do

Reply:
It can't be assessed yet though right? That's #2?